code/modules/mob/living/carbon/human/species/apollo.dm

```diff
@@ -0,0 +1,87 @@
+/datum/species/wryn
+	name = "Wryn"
+	icobase = 'icons/mob/human_races/r_wryn.dmi'
+	deform = 'icons/mob/human_races/r_wryn.dmi'
+	language = "Wryn Hivemind"
+	tail = "wryntail"
+	unarmed_type = /datum/unarmed_attack/punch/weak
+	primitive = /mob/living/carbon/monkey/wryn
+	darksight = 3
+	slowdown = 1
+	warning_low_pressure = -300
+	hazard_low_pressure = 1
+	blurb = "The wryn (r-in, singular r-in) are a humanoid race that possess many bee-like features. Originating from Alveare they \
+	have adapted extremely well to cold environments though have lost most of their muscles over generations.\
+	In order to communicate and work with multi-species crew Wryn were forced to take on names. Wryn have tended towards using only \
+	first names, these names are generally simplistic and easy to pronounce. Wryn have rarely had to communicate using their mouths, \
+	so in order to integrate with the multi-species crew they have been taught broken sol?."
+
+	cold_level_1 = 200 //Default 260 - Lower is better
+	cold_level_2 = 150 //Default 200
+	cold_level_3 = 115 //Default 120
+
+	heat_level_1 = 300 //Default 360 - Higher is better
+	heat_level_2 = 310 //Default 400
+	heat_level_3 = 317 //Default 1000
+
+	body_temperature = 286
+
+	has_organ = list(
+		"heart" =    /obj/item/organ/heart,
+		"brain" =    /obj/item/organ/brain,
+		"eyes" =     /obj/item/organ/eyes,
+		"appendix" = /obj/item/organ/appendix,
+		"antennae" =    /obj/item/organ/wryn/hivenode
+		)
+
+	flags = IS_WHITELISTED | HAS_LIPS | HAS_UNDERWEAR | NO_BREATHE | HAS_SKIN_COLOR | NO_SCAN | NO_SCAN | HIVEMIND
+
+
+	base_color = "#704300"
+	flesh_color = "#704300"
+	blood_color = "#FFFF99"
+
+
+/datum/species/wryn/handle_death(var/mob/living/carbon/human/H)
+	for(var/mob/living/carbon/C in world)
+		if(locate(/obj/item/organ/wryn/hivenode) in C.internal_organs)
```

Comment:
Eh, not a critical problem then.

You might want to swap the in world to a in living_mob_list or something though, just for a little bit of extra speed.